content/community.html

```diff
@@ -52,6 +52,17 @@ <h1>Contributing</h1>
       <a href="https://groups.google.com/forum/#!forum/prometheus-developers">prometheus-developers</a> – for discussions around Prometheus development
     </p>
 
+    <h1>Development summits</h1>
+    <p>
+      We strive to be as open as public as possible. Technical discussions happen on the development list, our in-person meeting notes are public, and we have public calls. Below, you can find links to our dev summits at PromCon 2017 & 2018, respectively. These dev summits, are open to join upon request as long as we have capacity. Preference is given to friendly projects and companies, plus diversity. Attendees are free not to be listed in the public meeting notes.
```

Comment:
"These dev summits," -> remove comma

I would spell out "developer summits" here, since "dev" is a bit informal.